### PR TITLE
[DOC, FIX] typos@include_seqan_align

### PR DESCRIPTION
**Pull request by beekalam**

### CODE REVIEW

File: include/seqan/align/fragment.h [hunk 1]
```diff
@@ -65,7 +65,7 @@ struct ExactFragment;
  * @headerfile <seqan/align.h>
  * @brief A type for ungapped, pairwise segment matches that maybe in reverse orientation.
  *
- * Compared to the @link ExactFragment @endlink specialzing type of @link Fragment @endlink, a @link
+ * Compared to the @link ExactFragment @endlink specialized type of @link Fragment @endlink, a @link
```
Comment (rrahn) [inline include/seqan/align/fragment.h]: Compared to the @link ExactFragment @endlink type specializing @link Fragment @endlink, a ...

**Comment by rrahn** [comment]
Hey @beekalam, thanks for fixing so many typos in our documentation. 
That is really great support for us 👍!!!
I'd like you to ask to gather all typos in a separate branch and push them all at once, if possible?
I guess, there will follow some more? 😞 Than we won't have them spread in so many PR's. 
Thanks a lot!

**Comment by beekalam** [comment]
thanks @rrahn.
I thought it would be more manageable at your end merge smaller branches. From now on, I'll hold the branch 3 or 4 days before making a pull request.

In fact I get confused myself.

Ironically the other day I accidentally pushed a branch to master. Sorry. 
If possible remove the branch. I will add changes at a later time in a different branch.

File: include/seqan/align/fragment.h [hunk 2]
```diff
@@ -65,7 +65,7 @@ struct ExactFragment;
  * @headerfile <seqan/align.h>
  * @brief A type for ungapped, pairwise segment matches that maybe in reverse orientation.
  *
- * Compared to the @link ExactFragment @endlink specialzing type of @link Fragment @endlink, a @link
+ * Compared to the @link ExactFragment @endlink <tt>ExactReverseable</tt> is a specialized type of @link Fragment @endlink. A @link
  * ExactReversableFragment @endlink stores an additional bool value to indicate whether a match is in reverse
```
Comment (rrahn) [inline include/seqan/align/fragment.h]: Should it be ExactReversableFragment?

Comment (beekalam) [inline include/seqan/align/fragment.h]: No
sorry my bad.
corrected.

**Comment by rrahn** [comment]
Can  be merged as soon as CLA has been signed.

**Pull request merged**